scripts/bib2md.py

```diff
@@ -0,0 +1,163 @@
+# Install the library before, with the following command:
+# pip install bibtexparser --pre
+
+# to run on the CLI:
+# python bib2md.py -f reference_file.bib -ord desc|asc
+
+import bibtexparser
+import sys
+import argparse
+
+def format_simple(entry_str, order):  
+    library = bibtexparser.parse_string(entry_str)
+    formatted_entries = []
+    unprocessed_entries = []
+    
+    for entry in library.entries:
+        try:
+            authors = entry['author'].split(' and ')
+            if len(authors) > 1:
+                authors[-1] = 'and ' + authors[-1]
+              
+            authors_formatted = ', '.join([a.replace('\n', ' ').strip() for a in authors])
+            title = entry['title']
+            year = int(entry['year'])
+            venue = entry.get('journal') or entry.get('booktitle') or entry.get('archivePrefix')
+            
+            if not venue:
+                id_unprocessed = "[" + entry.key + " - " + entry.entry_type + "]"
+                unprocessed_entries.append(id_unprocessed)
+                continue
+            
+            formatted_entries.append((year, f"{authors_formatted}. {title}. {venue.value}. ({year})."))
+
+        except KeyError as e:
+            print(f"One or more necessary fields {str(e)} not present in this BibTeX entry.")
+            continue
+
+    if order=='asc':
+        formatted_entries.sort(key=lambda x: x[0])
+    elif order=='desc':
+        formatted_entries.sort(key=lambda x: x[0], reverse=True)
+        
+    if len(unprocessed_entries) > 0:
+      print('Warning: Some entries were not processed due to unknown type', file=sys.stderr)
+      print("List of unprocessed entrie(s): ", unprocessed_entries)
+            
+    return [entry[1] for entry in formatted_entries]
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    
+    parser.add_argument('-f', '--file', type=str, 

```

Comment:
This must be an argument, not an option. If no argument is provided, the script should throw an error and terminate.